modes/preclinical-4d/src/index.tsx

```diff
@@ -101,39 +110,39 @@ function modeFactory({ modeConfiguration }) {
           id: 'dataPreparation',
           name: 'Data Preparation',
           hangingProtocol: {
-            protocolId: 'default4D',
+            protocolId: preclinical4d.hangingProtocol,
             stageId: 'dataPreparation',
           },
         },
         {
           id: 'registration',
           name: 'Registration',
           hangingProtocol: {
-            protocolId: 'default4D',
+            protocolId: preclinical4d.hangingProtocol,
```

Comment:
I thought in each step we are applying a new protocol no?

Reply:
hmmm this is confusing then and requires the a preexisting knowledge of the hanging protcol and that it should have multiple stages defined. Explicit code is much better in this situation so at least put a stage index or something as the argument

Reply:
I'm not sure if I'm following because all workflow stages are using the same `protocolId` but each one of them has a different `stageId`.

Reply:
Nevermind I understand now. Also when did we choose to add `workflow` to the mode root key? I thought we are using it in the onModeEnter and register the steps via the workflowService no?

Also seems like we have stages in the HP and stages in workflow which is a lot confusing we should think about how to rename the workflow stage to something else. I think we should opt in for `WorkflowStep` and `WorkflowStepService`

Reply:
Renamed "stages" to "steps" everywhere